reductus/web_gui/server_flask.py

```diff
@@ -50,6 +50,22 @@ def root():
             return redirect(posixpath.join(STATIC_FOLDER, DIST_PATH, CLIENT))
         else:
             return redirect(posixpath.join(STATIC_FOLDER, CLIENT))
+        
+    @app.route('/<instrument>')
+    def use_instrument(instrument):
+        # A simplified interface to load a specific instrument
+        # This uses the existing GET structure, but is a bit less strict on the name
+        known_instruments = api.list_instruments()
+        # Explicitly coerce the instrument name to a lower-case string
+        instrument = str(instrument).lower()
+        # Get base URL
+        initial_redirect = root()
+        if instrument not in known_instruments and config.get('instrument_prefix', None):
+            # Allow passed instrument name to be { vsans | sans | refl } instead of ncnr.{ vsans | sans | refl }
+            instrument = f"{config.get('instrument_prefix')}.{instrument}"
```

Comment:
Check if facility.instrument is known before assigning. This will reduce confusion when someone enters `ncnr.svans` and it comes out with `ncnr.ncnr.svans`. Or maybe just attempt the prefix if there is no '.' in instrument, so that `reductus/svans` will show up as a 404 for `ncnr.svans`.